### ISSUE TITLE
Image resizes inconsistently with other plugins

### ISSUE THREAD

**Issue by livestate-zz** [adeelejaz/jquery-image-resize#7]
Sorry if this has been reported before.

I have noticed that when I use the plugin with some other Jquery plugins the image does not display at all or is not resized. If the browser is then refreshed the image will display correctly and is resized corretly.

I have two sites that I am using the plugin with and with both of them the image resizing is inconsistent.

At www.thescarletbow.com.au the About page displays an image of the author. On first page load the image does not display. Refresh the browser and the image is then displayed.

At www.stillframe.com.au/index.php (index.php must be in the URL) the About page displays an imge of the author. On first page load the image is not resized but the overflow is hidden as Ive placed the image inside a constrained div with overflow hidden. On refresh the image displays resized.

Any thoughts?

**Comment by adeelejaz** [comment]
Can you please confirm you are using the latest version?
https://github.com/adeelejaz/jquery-image-resize/blob/master/jquery.ae.image.resize.js

**Comment by livestate-zz** [comment]
Im an idiot, seriously. 

I was just looking through the script and none of it looked right and I've just realised thats becasue its not your script I was using but another plugin by a different. 

I was using yours but the issue...the real issue...at the time was the jquery background image plugin was not working. Ironically, I just copied your script over to the site, changed the function and the backgorund image loads perfectly.

I'll keep using your plugin as it is a nicer version than the other one. I will double check the other site now as well and change the plugin to yours and confirm that works as intended.

Mark
